alacritty_terminal/src/url.rs

```diff
@@ -93,7 +93,9 @@ impl UrlParser {
                 ')' if open_parens_count > 0 => open_parens_count -= 1,
                 '[' => open_bracks_count += 1,
                 ']' if open_bracks_count > 0 => open_bracks_count -= 1,
-                ')' | ']' => {
+                '⟨' => open_bracks_count += 1,
+                '⟩' if open_bracks_count > 0 => open_bracks_count -= 1,
```

Comment:
These angle brackets should not be treated as the same as `[` in terms of nesting level.

Reply:
Got it.